frontend/integration-tests/views/kubernetes-marketplace.view.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-undef, no-unused-vars */
+
+import { browser, $, $$, ExpectedConditions as until } from 'protractor';
+
+export const entryTiles = $$('.catalog-tile-pf');
```

Comment:
Not sure what 'entry' stands for.  Can we call these `catalogTiles`? If so, I can use the same in my Catalog integration tests -thanks

Reply:
Sure thing!